### PR TITLE
chore(styles): add tokyonight inspired styles

### PR DESCRIPTION
**Pull request by cloudbridgeuy**
Add [`tokyonight`](https://github.com/folke/tokyonight.nvim) based styles.

`tokyonight-storm`
image
`tokyonight-day`
image
`tokyonight-moon`
image
`tokyonight-night`
image

### CODE REVIEW

**Comment by alecthomas** [comment]
"tokyonight-night" 😂

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/alecthomas/chroma/v2
+module github.com/cloudbridgeuy/chroma/v2
```
Comment (alecthomas) [inline go.mod:1]: Remove this.

**Review by alecthomas — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (cloudbridgeuy) [reply go.mod]: Yes. Sorry, I was trying to see how this would work on the `feeze` project.

**Comment by cloudbridgeuy** [comment]
> "tokyonight-night" 😂

Yeah. I didn't wanted to break the naming convention.

**Comment by alecthomas** [comment]
Awesome, thanks.

**Pull request merged**